wiki/People/Global_Moderation_Team/de.md

```diff
@@ -131,6 +127,7 @@ Die [Seite für das globale Moderationsteam](https://osu.ppy.sh/groups/4) listet
 | ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) | Philippinisch | Chat-Moderation |
 | ::{ flag=TH }:: [Trigonoculus](https://osu.ppy.sh/users/7627013) | Thailändisch | Chat-Moderation |
 | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | Spanisch | Technischer Support |
+| ::{ flag = NL }:: [Uni](https://osu.ppy.sh/users/617106) | Hollandisch | Chat-Moderation |
```

Comment:
```suggestion
| ::{ flag=NL }:: [Uni](https://osu.ppy.sh/users/617106) | Hollandisch | Chat-Moderation |
```

Reply:
osu-wiki translate-groups wasn't working on my laptop, going to try it on a different computer to in a little bit.